hw1/02-palindrome.js

```diff
@@ -1,3 +1,27 @@
 const elem = document.querySelector('input');
 
 elem.addEventListener('input', handleInput);
+output = document.getElementById("result")
+
+function handleInput() {
```

Comment:
Also, it was pointed out to me that instead of function declarations we should be using named function expressions since normal function declarations are hoisted. However, since you only have one function I don't think it's really a big issue here. [Airbnb style reference](https://github.com/airbnb/javascript#functions)

Reply:
fixed